src/client/I2PService.h

```diff
@@ -199,9 +199,7 @@ class TCPIPAcceptor : public I2PService {
   virtual std::shared_ptr<I2PServiceHandler> CreateHandler(
       std::shared_ptr<boost::asio::ip::tcp::socket> socket) = 0;
 
-  virtual std::string GetName() {
-    return "generic TCP/IP accepting daemon";
-  }
+  std::string GetName() const { return "generic TCP/IP accepting daemon"; }
```

Comment:
Just want to note that I agree that we should refrain from one-liners (and inline implementation overall).